### PR TITLE
Update dotnet1.0.1

### PR DESCRIPTION
**Pull request by rmarinho**
Just trying to make it build with the latest bits.. almost there

### CODE REVIEW

**Comment by neutmute** [comment]
nice work! is this all ready to merge? (Not sure based on the 'almost there' comment)
I'll pull this down and see if it builds at home anyway.

**Comment by rmarinho** [comment]
@neutmute can you pull this and run locally? i made it run, i don't know if does what is supposed, for example getting the ip is now async, and it messes with the initialization code on the Program.cs ..
My idea was if you can  pull and run and see if does what it did before.. then we can get it merged.

**Comment by neutmute** [comment]
no probs -will do

**Comment by neutmute** [comment]
It doesn't quite work for me when i ran it - seems some code isn't initialising. I'll just merge as it looks 99% there.

**Comment by neutmute** [comment]
solid work btw - thanks!

**Comment by neutmute** [comment]
Hey so the UDP multicast seems to work but the HTTP server isn't.

eg: On the RC-1 version you could run the server project and hit this URL say
`http://localhost/YamahaExtendedControl/v1/NetUsb/getPlayInfo`
and get a json response

With the latest version it doesn't seem to be listening.
That async IP code isn't helping - it returns null for me - although when i hardcoded the ip it still didn't listen.

If you can get this version to run and respond to 
`http://localhost/YamahaExtendedControl/v1/NetUsb/getPlayInfo`
then I think you should be good.

File: src/server/Program.cs [hunk 1]
```diff
@@ -30,16 +31,19 @@ public static void Main(string[] args)
...
-                //.Start();
+            var host = new WebHostBuilder()
+                .UseConfiguration(config)
+                .UseKestrel()
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseIISIntegration()
+                .UseStartup<Startup>()
+                .Build();
 
             Console.WriteLine($"Started the server. Listing on {uriToListenString}");
```
Comment (neutmute) [inline src/server/Program.cs:48]: This log should probably say something like
`Started the server. Listing on http://192.168.1.1:80;http://192.168.1.1:51123;http://192.168.1.1:51100`

 if the `MusicCastHost.ThisIp` is working

**Comment by rmarinho** [comment]
yeah the way to get the ip is async, and when you init you are using, we just have to change that code. I will take a look to night, btw and the JIG does what? Can you add that file to the repo that the JIG program is reading , if it+s important.

**Comment by neutmute** [comment]
I added more docs re: the Jig - should clear it up
https://github.com/neutmute/swimbait
https://github.com/neutmute/swimbait/tree/master/src/Jig

**Pull request merged**